### ISSUE TITLE
SecurityHub: Custodian Support for "Workflow" field mapping ASFF attributes

### ISSUE THREAD

**Issue by wallabyies** [cloud-custodian/cloud-custodian#5979]
**Is your feature request related to a problem? Please describe.**
The problem is that the post-finding action of Custodian cannot update the Workflow field of a SecurityHub finding.     

If a resource has a previous finding that is set to resolved, but has the finding later become active again for that resource, the finding remains in a resolved workflow state.    There currently seems to be no way for Custodian to change the Workflow value of a finding.

The Custodian documentation does not show the workflow field being mapped: https://cloudcustodian.io/docs/aws/topics/securityhub.html?highlight=security%20hub#id1

The AWS documentation shows that it is an available field:
https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-findings-format.html

**Describe the solution you'd like**
I would like the Workflow field to be mapped.    That way the post-finding action of a policy can set the Workflow value to NEW.

**Describe alternatives you've considered**
The only workaround I can see is filtering findings by "Updated at" in the SecurityHub console to show recent findings.     That way if a finding is resolved, it will show as recently updated.

**Comment by kapilt** [comment]
yeah.. the workflow field and ideological workflows of the product (these fields for end users, these fields for findings providers, these fields updatable and only by users is a master account not users in a member account or finding providers) is mostly a fluid mess on mapping, the rationale of which is unclear (don't get me started on the arbitrary attributes they choose to include in a finding from a resource). that said, workflow attribute should be exposed on the post-finding filter to enable this, thanks for filing the issue.

**Comment by wallabyies** [comment]
Thank you, I appreciate the help!
